BepInEx.IL2CPP/Preloader.cs

```diff
@@ -36,8 +40,7 @@ public static void Run()
                 }
 
                 ChainloaderLogHelper.PrintLogInfo(Log);
-
-                Log.LogInfo($"Running under Unity v{FileVersionInfo.GetVersionInfo(Paths.ExecutablePath).FileVersion}");
+                InitializeUnityVersion();
```

Comment:
If this is run here, Il2CppUnhollower runtime library will be initialized before unhollower DLLs are generated. This may cause issues as the unhollower was designed to work in in two steps:

1. Generate unhollowed assemblies
2. Initialize runtime library based on the generated unhollowed assemblies.

As such, this should be run after `ProxyAssemblyGenerator.GenerateAssemblies()` call.